kitchen-tests/Gemfile

```diff
@@ -6,5 +6,5 @@ gem "ohai", git: "https://github.com/chef/ohai.git", branch: "master" # avoids f
 gem "berkshelf", git: "https://github.com/berkshelf/berkshelf.git", branch: "master"
 gem "kitchen-dokken", "~> 2.0"
 gem "kitchen-inspec", git: "https://github.com/chef/kitchen-inspec.git", branch: "master"
-gem "inspec", git: "https://github.com/inspec/inspec.git", branch: "master" # this goes away when we ship inspec 4
-gem "test-kitchen", git: "https://github.com/test-kitchen/test-kitchen.git", branch: "master"
+gem "inspec"
+gem "test-kitchen", git: "https://github.com/tyler-ball/test-kitchen.git", branch: "test-tb"
```

Comment:
Let's make sure we have a checklist item to remove this once the test-kitchen change lands.